cf_mssql_broker_config.json

```diff
@@ -32,7 +32,7 @@
 	"brokerMssqlConnection": {
 		"server":   "localhost\\sqlexpress",
 		"database": "master",
-		"driver": 	"sql server",
+		"driver": 	"{SQL Server Native Client 11.0}",
```

Comment:
The default "sql server" odbc driver comes by default with windows and is pretty old ( https://www.connectionstrings.com/microsoft-sql-server-odbc-driver/info-and-download/ )
It looks like it's still maintained, but I think I won't support fancy features like connecting to a cluster, etc... 
Odbc driver version 11 comes with sql 2012 and sql 2014. I thought it would be best to bump this to that latest version at least for the example config file. 
The template for the installer is still untouched: https://github.com/cloudfoundry-incubator/cf-mssql-broker/blob/master/installer/cf_mssql_broker_config.json#L35

I could revert this change if you think this will create confusion / trouble.